modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Translation.js

```diff
@@ -40,9 +40,9 @@ export default function Translation({
 		timeSpanKey,
 	});
 
-	const timeSpanOffset = useMemo(() => {
-		return chartState.timeSpanOffset;
-	}, [chartState]);
+	const timeSpanOffset = useEffect(() => chartState.timeSpanOffset, [
```

Comment:
Why don't you just use `chartState.timeSpanOffset` [in place](https://github.com/liferay-frontend/liferay-portal/pull/287/commits/91adb76d29bac8dd3f6060dbf33e33fc81f03616#diff-eaff3b91e1be816f13bbea181d2bb6abR89)?

Reply:
OMG, you're right.